feature-group-definitions/subgrid.yml

```diff
@@ -2,13 +2,14 @@ spec: https://drafts.csswg.org/css-grid-2/#subgrids
 caniuse: css-subgrid
 usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/4680
 status:
-  is_baseline: true
-  since: "2023-09-15" # since Edge 117
+  baseline: false
```

Comment:
Shouldn't this be baseline low?

Reply:
This is "correct" because BCD says that the [`css.properties.grid-template-rows.subgrid` feature is unsupported on Firefox for Android](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-rows#browser_compatibility). This is "wrong" because the data is likely to be wrong (BCD says `css.properties.grid-template-columns.subgrid` _is_ supported on Firefox for Android).

Which is to say, the Baseline status is wrong for the right reason.

Reply:
I see https://github.com/mdn/browser-compat-data/pull/21388 should fix this - nice :+1:

Reply:
And I've pushed a change to accept the latest value now. 👍